UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Extensions/DateTimeExtensions.cs

```diff
@@ -13,5 +13,10 @@ public static bool IsValidDate(this string data, out DateTime dateTime)
         {
             return DateTime.TryParseExact(data, "ddMMMyyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out dateTime);
         }
+        public static int GetValidTillDays(int validPastWeeks)
```

Comment:
This method name is very confusing. As it is simply a multiplication by 7, does it really need a separate method?  Also I have suggest that the config value be 28 days rather than 4 weeks, which removes the need for this method.

Reply:
Yes, After configuring it as days no need of this hence removed it.